background.js

```diff
@@ -137,18 +136,58 @@ async function setupContainer () {
   }
 }
 
-async function containFacebook (options) {
-  // Listen to requests and open Facebook into its Container,
-  // open other sites into the default tab context
-  const requestUrl = new URL(options.url);
+function reopenTab ({url, tab, cookieStoreId}) {
+  if (tab.incognito) {
+    // containers don't work incognito
+    return;
+  }
+
+  browser.tabs.create({
+    url,
+    cookieStoreId,
+    active: tab.active,
+    index: tab.index,
+    windowId: tab.windowId
+  });
+  browser.tabs.remove(tab.id);
+}
 
-  let isFacebook = false;
+function isFacebookURL (url) {
+  const parsedUrl = new URL(url);
   for (let facebookHostRE of facebookHostREs) {
-    if (facebookHostRE.test(requestUrl.host)) {
-      isFacebook = true;
+    if (facebookHostRE.test(parsedUrl.host)) {
+      return true;
       break;
```

Comment:
Wouldn't the return exit the function? is `break` required?

Reply:
That's correct - was flying a bit blind without eslint. Fixed in #82